utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/DBUnitUtility.java

```diff
@@ -1,635 +1,620 @@
+    public static void loadDatabase(final String targetSchema, final String filePath, final boolean reloadRefdata,
+                                    final boolean cleanseDatabase) {
+
+        try {
+
+            // database connection
+            LOGGER.info("Starting load for schema[" + targetSchema + "], filePath[" + filePath + "], reloadRefdata[" +
+                    reloadRefdata + "]");
+            final IDatabaseConnection dbConnection = getConnectionInstance(targetSchema);
+
+            // Create a DBUnit Dataset builder using the target schemas DTD
+            final FlatXmlDataSetBuilder builder = new FlatXmlDataSetBuilder();
+            builder.setDtdMetadata(true);
+            builder.setMetaDataSetFromDtd(DBUnitUtility.class.getResourceAsStream("/DTD/" + targetSchema + ".dtd"));
+            builder.setColumnSensing(true);
+
+            // Generate the clean dataset, if loadRefData is true use
+            // clean_all.xml otherwise use clean.xml
+            final String cleanDataPath = "/clean/" + targetSchema + "_clean_minus_refdata" + ".xml";
+            LOGGER.info("Reading clean dataset for data path=" + cleanDataPath);
+            final IDataSet cleanDataset = builder.build(DBUnitUtility.class.getResourceAsStream(cleanDataPath));
+
+            final IDataSet dtdDataSet =
+                    new FlatDtdDataSet(DBUnitUtility.class.getResourceAsStream("/DTD/" + targetSchema + ".dtd"));
+            final CompositeDataSet compositeDataSet = new CompositeDataSet(dtdDataSet, cleanDataset);
+
+            // Disable triggers & constraints
+            prepareForDbunitLoad(targetSchema);
+
+            if (cleanseDatabase) {
+                // Clean the DB
+                LOGGER.info("Cleaning database");
+                // DatabaseOperation.TRUNCATE_TABLE.execute (dbConnection, cleanDataset);
+                DatabaseOperation.TRUNCATE_TABLE.execute(dbConnection, compositeDataSet);
+                dbConnection.getConnection().commit();
+            }
+
+            // If there's reference data insert it
+            final String refDataPath = "/refdata/" + targetSchema + "_refdata.xml";
+            final InputStream refDataStream = DBUnitUtility.class.getResourceAsStream(refDataPath);
+            if (refDataStream != null && reloadRefdata) {
+                LOGGER.info("Inserting reference data from " + refDataPath);
+                final IDataSet referenceDataset = builder.build(refDataStream);
+                DatabaseOperation.INSERT.execute(dbConnection, referenceDataset);
+                dbConnection.getConnection().commit();
+            }
+
+            // Insert the test data
+            LOGGER.info("Inserting data from filePath=" + filePath);
+            final IDataSet targetDataset = buildTargetDataset(builder, filePath);
+
+            DatabaseOperation.INSERT.execute(dbConnection, targetDataset);
+
+            // Enable triggers & constraints
+//            finishDbunitLoad(targetSchema);
```

Comment:
Don't forget to remove this if it's no longer needed

Reply:
keepiing these for now, uncommented it now.
will delete DBUnitUtility.java, once we are happy with the changes done and having working application